our-work/software/xnat-tools.yml

```diff
@@ -0,0 +1,18 @@
+title: XNAT Tools
+description: "Python library that facilitates the export and conversion of data stored in Brown's XNAT platform to the Brain Imaging Data Structure (BIDS)"
+languages:
+  - python
+tags:
```

Comment:
on the current version, `CCV`, `CBC` and `DISCO` are our tags - should this be somewhere else? or is this also a tag? or maybe `groups`?